connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -651,6 +651,7 @@ class SparkConnectPlanner(session: SparkSession) {
         .map(transformExpression)
         .map {
           case x @ UnresolvedAttribute(_) => x
+          case x @ Alias(_, _) => x
```

Comment:
I think it would also make sense to add a test on scala side given this touches server.

Reply:
I'd like to do that as a follow up.